homeassistant/components/home_connect/sensor.py

```diff
@@ -195,28 +195,76 @@ class HomeConnectSensorEntityDescription(
 
 EVENT_SENSORS = (
     HomeConnectSensorEntityDescription(
-        key=EventKey.REFRIGERATION_FRIDGE_FREEZER_EVENT_DOOR_ALARM_FREEZER,
+        key=EventKey.BSH_COMMON_EVENT_PROGRAM_ABORTED,
```

Comment:
Can we sort the sensors in a separate PR where we don't add anything new? It's hard to review this when there's both new sensors and sensors that have just moved.